stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -734,15 +766,17 @@ extension ThrowingTaskGroup: AsyncSequence {
     return Iterator(group: self)
   }
 
-  /// Allows iterating over results of tasks added to the group.
+  /// A type that provides an iteration interface
+  /// over the results of tasks added to the group.
   ///
-  /// The order of elements returned by this iterator is the same as manually
-  /// invoking the `group.next()` function in a loop, meaning that results
-  /// are returned in *completion order*.
+  /// The elements returned by this iterator
+  /// appear in the order that the tasks *completed*,
+  /// not in the order that those tasks were added to the task group.
```

Comment:
well worded! Can we add the same to the next() and nextResult() on the group itself?

Reply:
Where else did you want to add this?  I've checked the `next` and `nextResult` methods, and I think it's already on all of them.